### DISCUSSION TITLE
Is it possible to move  nvchad snapshot to other device?

### DISCUSSION THREAD

**Discussion by KirovAI** [NvChad/NvChad#1361] [Q&A] [answered]
I have several devices on which i use neovim and I not want perform each neovim.
Maybe possible make one snapshot  and just move on other device?
In advance, thanks

**Comment by siduck** [comment, marked as answer]
Soon @LeonHeidelbach will implement this ⚡

**Reply by LeonHeidelbach** [reply]
I'll start working on this the week after the next one ;)
